MetamodelAlgemeen.md

```diff
@@ -268,7 +268,7 @@ Om duidelijk(er) te maken wat wordt bedoeld kijken we eerst naar het begrip ‘o
 #### Object
 
 <aside class="definition">
-  <dfn data-lt="objecten">Object</dfn>Een ding, een tastbaar iets, in de werkelijkheid, zoals daarnaar gekeken wordt vanuit een bepaald domein.
+  <dfn data-lt="objecten">Object</dfn>Een object is een identificeerbaar iets in het beschouwde domein.
```

Comment:
```suggestion
  <dfn data-lt="objecten">Object</dfn>Een object is een onderscheidbaar iets in het beschouwde domein.
```

Identificeerbaar vind ik erg lastig. Dit doet denken dat elk object en dus elk objecttype een identificeerbaar kenmerk moet hebben in een logisch gegevensmodel. Dat is niet zo. Er kunnen prima objecten zijn die we niet willen of kunnen identificeren.
Ik stel voor om identificeerbaar te vervangen met onderscheidbaar.